packages/core/src/external/surescripts/command/bundle/get-bundle.ts

```diff
@@ -0,0 +1,27 @@
+import { Bundle, BundleEntry } from "@medplum/fhirtypes";
+import { buildCollectionBundle } from "../../../fhir/bundle/bundle";
+
+/**
+ * TODO ENG-476 Implement this
+ */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+export async function getPharmacyBundle({
+  cxId,
+  patientId,
+}: {
+  cxId: string;
+  patientId: string;
+}): Promise<Bundle> {
+  return buildCollectionBundle([]);
```

Comment:
Assuming you rebase with the develop branch, can we implement this right away?

```
import { buildLatestConversionBundleFileName } from "../../../file/file-names";

// ...

const s3Utils = new S3Utils(Config.getAWSRegion());
const bucketName = Config.getPharmacyConversionBucketName();
const fileName = buildLatestConversionBundleFileName(cxId, patientId);
if (! await s3Utils.fileExists(bucketName, fileName))
  return buildCollectionBundle([]);
return await s3Utils.downloadFile(bucketName, fileName)
```

Reply:
I'll tackle it on a diff PR while this ships